Production/govuk_ios/ServiceClients/TopicsServiceClient.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+
+typealias FetchTopicsListResult = (Result<[Topic], TopicsListError>) -> Void
+
+protocol TopicsServiceClientInterface {
+    func fetchTopicsList(completion: @escaping FetchTopicsListResult)
+}
+
+enum TopicsListError: Error {
+    case apiUnavailable
+    case decodingError
+}
+
+struct TopicsServiceClient: TopicsServiceClientInterface {
+    private let serviceClient: APIServiceClientInterface
+
+    init(serviceClient: APIServiceClientInterface) {
+        self.serviceClient = serviceClient
+    }
+
+    func fetchTopicsList(completion: @escaping FetchTopicsListResult) {
+        let topicsRequest = GOVRequest(
+            urlPath: "/static/topics/list",
+            method: .get,
+            bodyParameters: nil,
+            queryParameters: nil,
+            additionalHeaders: nil
+        )
+
+        serviceClient.send(
+            request: topicsRequest,
+            completion: { result in
+                switch result {
+                case .success(let data):
+                    do {
+                        let topics = try JSONDecoder().decode([Topic].self, from: data)
+                        completion(.success(topics))
```

Comment:
I made this comment on another PR https://github.com/alphagov/govuk-mobile-ios-app/pull/79/files#r1768737887 and Tom said there will be future change where APIServiceClient will no longer call send completion on main thread. Mentioning incase this needs to be considered here?